config/locales/en.yml

```diff
@@ -22,12 +22,101 @@
 en:
   activerecord:
     attributes:
-      address:
+      address: &address_attributes
+        address_1: 'Street (Line 1)'
+        address_2: 'Street (Line 2)'
+        city: 'City'
+        country: 'Country Code'
+        postal_code: 'ZIP Code'
         state_province: 'State'
+
+      api_application:
+        name: 'Application Name'
+        main_url: 'Main URL'
+        callback_url: 'Callback URL'
+
+      contact:
+        email: 'Contact Email'
```

Comment:
Looking at this diff now, it may make more sense to move these "Contact FIELD" strings to the `helpers.labels` namespace. I think this could go for the `website` localizations as well.